aspnetcore/performance/caching/middleware.md

```diff
@@ -12,7 +12,7 @@ uid: performance/caching/middleware
 
 By [Luke Latham](https://github.com/guardrex) and [John Luo](https://github.com/JunTaoLuo)
 
-[View or download ASP.NET Core 2.1 sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/middleware/samples) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+[View or download ASP.NET Core 2.1 sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/middleware/samples) ([how to download](xref:index#how-to-download-a-sample))
```

Comment:
I recommend the "View or download sample code" link text here too (for the sake of consistency).